src/three/plugins/SUBTREELoader.js

```diff
@@ -0,0 +1,747 @@
+			currentRow = [];
+		}
+
+		return parentRow;
+
+	}
+
+	/**
+	 * Given a parent tile and information about which child to create, derive
+	 * the properties of the child tile implicitly.
+	 * <p>
+	 * This creates a real tile for rendering.
+	 * </p>
+	 *
+	 * @param {Subtree} subtree The subtree the child tile belongs to
+	 * @param {Object | SubtreeTile} parentTile The parent of the new child tile
+	 * @param {number} childBitIndex The index of the child tile within the tile's availability information.
+	 * @param {number} childMortonIndex The morton index of the child tile relative to its parent
+	 * @returns {SubtreeTile} The new child tile.
+	 * @private
+	 */
+	deriveChildTile(
+		subtree,
+		parentTile,
+		childBitIndex,
+		childMortonIndex
+	) {
+		let subtreeTile = new SubtreeTile(parentTile, childMortonIndex);
+		subtreeTile.boundingVolume = this.getTileBoundingVolume(subtreeTile);
+
+		subtreeTile.geometricError = this.getGeometricError(subtreeTile);
+
+		// Todo Multiple contents not handled, keep the first found content
+		for (let i = 0; subtree && i < subtree._contentAvailabilityBitstreams.length; i++) {
+			if (subtree && this.getBit(subtree._contentAvailabilityBitstreams[i], childBitIndex)) {
+				subtreeTile.content = {uri: this.parseImplicitURI(subtreeTile, this.rootTile.__basePath, this.rootTile.__contentUri)};
+				break;
+			}
+		}
+
+		return subtreeTile;
+	}
+
+
+	/**
+	 * Get a bit from the bitstream as a Boolean. If the bitstream
+	 * is a constant, the constant value is returned instead.
+	 *
+	 * @param {ParsedBitstream} object
+	 * @param {number} index The integer index of the bit.
+	 * @returns {boolean} The value of the bit
+	 * @private
+	 */
+	getBit(object, index) {
+		if (index < 0 || index >= object.lengthBits) {
+			throw new Error("Bit index out of bounds.");
+
+		}
+
+		if (object.constant !== undefined) {
+			return object.constant;
+		}
+
+
+		// byteIndex is floor(index / 8)
+		const byteIndex = index >> 3;
+		const bitIndex = index % 8;
+
+		return ((new Uint8Array(object.bitstream)[byteIndex] >> bitIndex) & 1) === 1;
+	}
+
+	/**
+	 * //TODO Adapt for Box & Sphere
+	 * To maintain numerical stability during this subdivision process,
+	 * the actual bounding volumes should not be computed progressively by subdividing a non-root tile volume.
+	 * Instead, the exact bounding volumes are computed directly for a given level.
+	 * @param {Object | SubtreeTile} tile
+	 * @return {Object} object containing the bounding volume
+	 */
+	getTileBoundingVolume(tile) {
+		let boundingVolume;
+		if (this.rootTile.boundingVolume.region) {
+			boundingVolume = [...this.rootTile.boundingVolume.region];
+			const minX = boundingVolume[0];
+			const maxX = boundingVolume[2];
+			const minY = boundingVolume[1];
+			const maxY = boundingVolume[3];
+			const sizeX = (maxX - minX) / (Math.pow(2, tile.__depth ?? tile.__level));
+			const sizeY = (maxY - minY) / (Math.pow(2, tile.__depth ?? tile.__level));
```

Comment:
I think I mentioned this before but there shouldn't be any reason to use `__depth` here. `__level` is the more correct value to use, no?